R/seroprevalence_data.R

```diff
@@ -120,4 +120,203 @@ prepare_bin_data <- function(serodata) {
     p_obs_bin_u = .data$Upper
   )
   return(xx)
+}
+
+#' Function that generates the probabilities of being previously exposed to a pathogen.
+#'
+#' @param sim_data A dataframe object containing the following columns:
+#' \tabular{ll}{
+#' \code{birth_year} \tab List of years in which the subjects were borned \cr \tab \cr
+#' \code{tsur} \tab Year of the survey\cr \tab \cr
+#' \code{country} \tab Default to 'none'.\cr \tab \cr
+#' \code{survey} \tab Survey label \cr \tab \cr
+#' \code{age_mean_f} \tab Age \cr \tab \cr
```

Comment:
It is redundant but a practical reason for this is that `get_exposure_ages()` uses `serodata$birth_year` to generate the exposition ages, meanwhile `get_exposure_matrix()` uses the age-class marker `age_mean_f` to generate the exposure matrix. 

We may be able to remove this redundancy, but I suggest we do it in a separate PR because we need to discuss first whether it's worth it to generate `birth_year` by means of `prepare_serodata` or not and refactor the package accordingly.